### PR TITLE
Fix/#129 remove confusing message

### PR DESCRIPTION
**Pull request by grozdanov**

### CODE REVIEW

**Comment by grozdanov** [comment]
Here is the test case for the updated `console.log()`: https://github.com/magic-script/magic-script-cli/blob/2131ea56a1c0702d1599932418ae67340ad76365/__tests__/build.js#L354

File: commands/build.js [hunk 1]
```diff
@@ -83,18 +83,19 @@ function buildLumin (argv, indexContent) {
...
-          break;
-        }
+
+      let theLine = outLines.find(line => line.includes('mpk'));
+      if (theLine !== undefined) {
+        mpkFile = theLine.substring(theLine.indexOf("'") + 1, theLine.lastIndexOf("'"));
+        console.log('built package: ' + mpkFile);
+      } else {
+        theLine = outLines.find(line => line.includes('output in'));
+        console.log(theLine.substring(theLine.indexOf("'"), theLine.lastIndexOf("'") + 1));
```
Comment (kpiascik) [inline commands/build.js]: what does this log line produce?  Can you provide me an example?

Comment (grozdanov) [reply commands/build.js]: 'app' output in '.out/app'

Comment (grozdanov) [reply commands/build.js]: The line informs the developer where the project has been built.

**Pull request merged**